pkg/Microsoft.ML.HalLearners/Microsoft.ML.HalLearners.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET additional learners making use of hardware acceleration. They depend on the MlNetMklDeps NuGetpackage. </PackageDescription>
```

Comment:
>NuGetpackage.  [](start = 120, length = 14)

wouldbenicetousespace

Reply:
one shall not space. 

---
In reply to: [206717086](https://github.com/dotnet/machinelearning/pull/611#discussion_r206717086) [](ancestors = 206717086)

Reply:
why it's still NuGetpackage?

---
In reply to: [206717405](https://github.com/dotnet/machinelearning/pull/611#discussion_r206717405) [](ancestors = 206717405,206717086)

Reply:
ah, i thought you were pointing out the space after the period. 
Fixed. 

---
In reply to: [206718135](https://github.com/dotnet/machinelearning/pull/611#discussion_r206718135) [](ancestors = 206718135,206717405,206717086)